sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -314,6 +314,32 @@ case class Divide(left: Expression, right: Expression) extends DivModLike {
   override def evalOperation(left: Any, right: Any): Any = div(left, right)
 }
 
+@ExpressionDescription(
+  usage = "expr1 _FUNC_ expr2 - Returns `expr1`/`expr2`. It performs integral division.",
```

Comment:
Let's mention that it always return long. Maybe we can take a look at how Hive document it.

Reply:
`Divide a by b rounded to the long integer`, this is Hive's `div` document.

Reply:
yes, thanks @viirya, I am updating to that sentence